src/AlertFlags/AlertFlags.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './AlertFlags.scss'
+
+const AlertFlags = ({ alerts, category }) => {
+  function isShown(code) {
+    if (alerts) {
```

Comment:
Is this correct here?  Can't be `falsey` since it would have blown up earlier with the `alerts.length`